scalableminds/webknossos
# Isosurfaces for whole segments

### URL of deployed dev instance (used for testing):
- https://isosurfaceneighbors.webknossos.xyz

### Steps to test:
- open a dataset with a segmentation in "view mode"
- enable isosurface rendering in the dataset settings
- shift click on a segment to initiate the isosurface rendering
- ideally, click on a segment which is relatively large so that its computation is split into chunks (parts should appear sequentially)

### Issues:
- fixes #3647 
- fixes #3688
------
